selftests/functional/test_nrunner.py

```diff
@@ -49,48 +48,6 @@ def test_noop(self):
         self.assertIn(b"'time': ", res.stdout)
         self.assertEqual(res.exit_status, 0)
 
-    def test_exec(self):
```

Comment:
I don't see a similar coverage for `exec-test`, which IMO would justify removing those.  So, instead of removing those, you should adapt them to `exec-test`, ie:

```diff
diff --git a/selftests/functional/test_nrunner.py b/selftests/functional/test_nrunner.py
index c81011171..b7fcabd3b 100644
--- a/selftests/functional/test_nrunner.py
+++ b/selftests/functional/test_nrunner.py
@@ -49,10 +49,10 @@ class RunnableRun(unittest.TestCase):
         self.assertIn(b"'time': ", res.stdout)
         self.assertEqual(res.exit_status, 0)
 
-    def test_exec(self):
+    def test_exec_test(self):
         # 'base64:LWM=' becomes '-c' and makes Python execute the
         # commands on the subsequent argument
-        cmd = ("%s runnable-run -k exec -u %s -a 'base64:LWM=' -a "
+        cmd = ("%s runnable-run -k exec-test -u %s -a 'base64:LWM=' -a "
                "'import sys; sys.exit(99)'" % (RUNNER, sys.executable))
         res = process.run(cmd, ignore_status=True)
         self.assertIn(b"'status': 'finished'", res.stdout)
@@ -60,9 +60,9 @@ class RunnableRun(unittest.TestCase):
         self.assertEqual(res.exit_status, 0)
 
     @skipUnlessPathExists('/bin/sh')
-    def test_exec_echo(self):
+    def test_exec_test_echo(self):
         # 'base64:LW4=' becomes '-n' and prevents echo from printing a newline
-        cmd = ("%s runnable-run -k exec -u /bin/echo -a 'base64:LW4=' -a "
+        cmd = ("%s runnable-run -k exec-test -u /bin/echo -a 'base64:LW4=' -a "
                "_Avocado_Runner_" % RUNNER)
         res = process.run(cmd, ignore_status=True)
         self.assertIn(b"'status': 'finished'", res.stdout)
@@ -104,7 +104,7 @@ class RunnableRun(unittest.TestCase):
 
     @skipUnlessPathExists('/bin/env')
     def test_exec_kwargs(self):
-        res = process.run("%s runnable-run -k exec -u /bin/env X=Y" % RUNNER,
+        res = process.run("%s runnable-run -k exec-test -u /bin/env X=Y" % RUNNER,
                           ignore_status=True)
         self.assertIn(b"'status': 'finished'", res.stdout)
         self.assertIn(b"X=Y\\n", res.stdout)

```